.gitignore

```diff
@@ -1,6 +1,6 @@
-*.aux
-*.log
-*.out
+**.aux
+**.log
+**.out
```

Comment:
*.Dateiendung wird doch schon immer in jedem Ordner gematcht? Wenn man will, dass eine Dateiendung nur im obersten Ordner ignoriert wird, dann muss man "/*.abc" nehmen